libgnucash/engine/Transaction.c

```diff
@@ -3100,5 +3100,106 @@ _utest_trans_fill_functions (void)
     return func;
 }
 
+static void
+record_price(Split *split,
+             PriceSource source)
+{
+    Transaction *trans;
+    Account *account;
+    QofBook* book;
+    GNCPriceDB* pricedb;
+    gnc_commodity* comm;
+    gnc_commodity* curr;
+    GNCPrice* price;
+    gnc_numeric price_value, value, amount;
+    int scu;
+    time64 time;
+    gboolean swap;
+
+    account = xaccSplitGetAccount(split);
+    if (!xaccAccountIsPriced(account)) {
+       return;
+    }
```

Comment:
See https://wiki.gnucash.org/wiki/CodingStandard - no dangling `{` `}` and a space before `(`

```c
    if (!xaccAccountIsPriced (account))
    {
       return;
    }
```

Reply:
fixed.